CONTRIBUTING.md

```diff
@@ -16,6 +16,111 @@ We are also present on the `#openfeature` channel in the [CNCF slack](https://sl
 
 Project governance is documented [here](https://github.com/open-feature/community/blob/main/governance-charter.md).
 
+## How to contribute
+
+All the OpenFeature repositories follow these guidelines.
+
+### Issues
+
+Issues are the heartbeat of open source projects. There are mainly three kinds of issues you can open:
+
+* Bug : If you believe you found a problem in a project, and you want to discuss and get it fixed,
+  creating an issue with the **bug template** is the best way to do so.
+* Documentation: Do want to improve the documentation? did you find a typo? This is the issue for you.
+* Feature: Would you like a new feature to be added to OpenFeature? This is the kind of issue you'll need then.
+  Please do your best at explaining your intent. It's always important that others can understand what you mean in order to discuss.
+  Be open and collaborative in letting others help you get things done!
+* Report a security vulnerability: Did you find a vulnerability? This issue kind is for triaging such problems!
+
+The best way to **get involved** in the project is through issues. You can help in many ways:
+
+* Issues triaging: participating in the discussion and adding details to open issues is always a good thing;
+sometimes issues need to be verified, you could be the one writing a test case to fix a bug!
+* Helping to resolve the issue: you can help in getting it fixed in many ways, more often by opening a pull request.
+
+### Triaging
+
+We need help in categorizing issues. Thus any help is welcome!
+
+When you triage an issue, you:
+
+* assess whether it has merit or not
+
+* quickly close it by correctly answering a question
+
+* point the reporter to a resource or documentation answering the issue
+
+* tag it via labels, projects, or milestones
+
+* take ownership submitting a PR for it, in case you want
+
+#### More about labels
+
+These guidelines are not set in stone and are subject to change.
+
+Each repository has its own set of labels. Here is the current [label set](https://github.com/open-feature/spec/labels) for the spec repository.
```

Comment:
Should we define a set of labels available in each repo ?

Reply:
right now we have different labels for each repo, we could try to align them. I'm not sure what the effort would be

Reply:
I would like to align. I wonder if we can do this in peribolos... the challenge is it seems like when you create org labels they are only created in subsequent new repos :(